redhat-appstudio/service-provider-integration-operator
# Svpi137 kcp vault

### What does this PR do?
Prepare Vault and SPI configuration to use external Ingress/Route to connect to Vault instead of service.

`make deploy_minikube` now deploys Vault in it's own namespace `spi-vault`

relates to #182. It won't work together as #182 is changing config file structure

depends on https://github.com/redhat-appstudio/service-provider-integration-oauth/pull/119

### Screenshot/screencast of this PR



### What issues does this PR fix or reference?

https://issues.redhat.com/browse/SVPI-137

### How to test this PR?

deploy SPI way of your choice. It must work, SPI operator and oauth must start and connect to Vault. You must see deployed Ingress/Route and `vaultHost` in configuration secret with vault url.
